packages/frontend/src/composables/useNoteModal.ts

```
@@ -163,4 +204,4 @@ export function useNoteModal(options: NoteModalOptions = {}) {
     save,
     initialize,
   };
-}
+}
```

ESLint

[error] Insert `⏎`

(prettier/prettier)